service/assessment/assessment_test.go

```diff
@@ -200,46 +219,27 @@ func TestService_AssessEvidences(t *testing.T) {
 				t.Errorf("AssessEvidence() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
+
+			if err != nil {
+				assert.Contains(t, err.Error(), tt.wantErrMessage)
+			}
 		})
 	}
 }
 
 func TestAssessmentResultHooks(t *testing.T) {
 	var (
 		hookCallCounter = 0
-		// Service needs to outlive the lifetime of the hook function
-		service = *orchestrator_service.NewService()
 	)
 
 	firstHookFunction := func(assessmentResult *assessment.AssessmentResult, err error) {
 		hookCallCounter++
-		log.Println("Hello from inside the firstHookFunction")
+		fmt.Println("Hello from inside the firstHookFunction")
```

Comment:
Are we logging or printing to stdout in tests?

Reply:
It depends. We use logging, printing and the log method of the testing package.

Reply:
I think we should use the log method of the testing package and if it is not possible use logging.